### ISSUE TITLE
multipart/form-data issue

### ISSUE THREAD

**Issue by sarangsbabu367** [p1c2u/openapi-core#231]
I have a data with content-type multipart/form-data, but in my request content-type is like, `multipart/form-data; boundary=---{some value}---` and the actual value reside in key,
`multipart/form-data` so openapi-core is raising error, content for `multipart/form-data; boundary=---{some value}---` not found.

**Comment by sarangsbabu367** [comment]
{
    "errors": [
        {
            "title": "Unknown",
            "detail": "Content for following mimetype not found: multipart/form-data; boundary=--------------------------946618265055152913063555"
        }
    ]
}

**Comment by sarangsbabu367** [comment]
That was an issue with content-type in spec. Not an openapi-core issue.

**Comment by remram44** [comment]
I'm running into this. How did you fix this in your spec? Just stopped using `multipart/form-data`?

**Comment by jgod** [comment]
How did you resolve this? What does this section of a working spec look like?

**Comment by remram44** [comment]
I had to drop the `type: object` from the top-level response definitions. I was also running into "schema is not a Schema" errors that I don't remember how I fixed.

[Here is my spec](https://github.com/VIDA-NYU/auctus/blob/53138d898a3b0ebfe60cd8e915b4b5ca742e1b5a/docs/schemas/restapi.yaml), though it references JSON Schema files that [I inline into it](https://github.com/VIDA-NYU/auctus/blob/53138d898a3b0ebfe60cd8e915b4b5ca742e1b5a/tests/utils.py#L18-L87) before using it with openapi-core (moving JSON Schema "definitions" to top-level OpenAPI "components/schemas" and removing top-level "type: object" entries).
